pyrefly/lib/solver/solver.rs

```diff
@@ -1238,24 +1253,38 @@ impl<'a, Ans: LookupAnswer> Subset<'a, Ans> {
                         let t2 = t2.clone();
                         drop(v2_ref);
                         drop(variables);
-                        self.is_subset_eq(t1, &t2)
+                        match self.is_subset_eq(t1, &t2) {
```

Comment:
(same thing here - I don't think we would ever want to change an answer after it has been decided already)